src/sql-parser/src/ast/defs/expr.rs

```diff
@@ -735,3 +743,22 @@ impl<T: AstInfo> AstDisplay for FunctionArgs<T> {
     }
 }
 impl_display_t!(FunctionArgs);
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum IsExprConstruct {
+    NULL,
+    TRUE,
+    FALSE,
+    UNKNOWN,
```

Comment:
Rust style is to use UpperCamelCase for enum variants rather than SNAKE_CASE.

Reply:
ahhhh it's been too long since I've written rust -- thanks